packages/core/contracts/zodiac/OptimisticGovernor.sol

```diff
@@ -192,23 +189,35 @@ contract OptimisticGovernor is Module, Lockable {
         // proposalHashes[id] = keccak256(abi.encodePacked(proposalData));
         proposalHashes[id] = keccak256(abi.encode(_transactions));
 
+        // Propose a set of transactions to the OO. If not disputed, they can be executed with executeProposal().
+        // docs: https://github.com/UMAprotocol/protocol/blob/master/packages/core/contracts/oracle/interfaces/OptimisticOracleInterface.sol
+
+        // optimisticOracle.requestAndProposePriceFor(
+        //     identifier,
+        //     uint32(time),
+        //     ancillaryData,
+        //     collateral,
+        //     0,
+        //     bond,
+        //     uint256(liveness),
+        //     address(this),
+        //     // Canonical value representing "True"; i.e. the transactions are valid.
+        //     int256(1e18)
+        // );
+        optimisticOracle.requestPrice(identifier, uint32(time), ancillaryData, collateral, 0);
+
+        uint256 totalBond = optimisticOracle.setBond(identifier, uint32(time), ancillaryData, bond);
```

Comment:
nit: you probably also need to make a call to set the liveness for this request.